elevenlabs_mcp/server.py

```
@@ -1215,6 +1216,97 @@ def create_composition_plan(
+
+    Returns:
+        TextContent containing the raw JSON from the API
+    """
+    if tool_id == "":
+        make_error("Tool ID is required.")
+    try:
+        response = client.conversational_ai.tools.get_dependent_agents(
+            tool_id=tool_id, cursor=cursor, page_size=page_size
+        )
+        return TextContent(type="text", text=response.model_dump_json(indent=2))
+    except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)